nuxeo-core/nuxeo-core-mongodb/src/test/java/org/nuxeo/ecm/core/mongodb/seqgen/TestSequenceGeneratorWithMongoDB.java

```diff
@@ -64,17 +74,57 @@ public void testIncrement() throws Exception {
     }
 
     @Test
+    @LogCaptureFeature.FilterOn(logLevel = "WARN")
     public void testInitSequence() {
         UIDSequencer seq = uidGeneratorService.getSequencer();
+        assertTrue(seq.getClass().isAssignableFrom(MongoDBUIDSequencer.class));
```

Comment:
That would be `MongoDBUIDSequencer.class.isAssignableFrom(seq.getClass())` but can simply be written as `seq instanceof MongoDBUIDSequencer` although I don't know why this needs to be tested.